subsys/bootloader/image/CMakeLists.txt

```diff
@@ -95,8 +86,8 @@ add_custom_command(
   COMMAND
   ${PYTHON_EXECUTABLE}
   ${NRF_BOOTLOADER_SCRIPTS}/provision.py
-  --s0-addr ${s0_addr}
-  ${s1_arg}
+  --s0-addr $<TARGET_PROPERTY:partition_manager,PM_S0_ADDRESS>
+  --s1-addr $<TARGET_PROPERTY:partition_manager,PM_S1_ADDRESS>
```

Comment:
You could also edit provision.py to make `--s1-addr` required.

Reply:
good point